components.json

```diff
@@ -579,6 +579,10 @@
 			"require": "json",
 			"owner": "RunDevelopment"
 		},
+		"jsstacktrace": {
+			"title": "Javascript stack trace",
```

Comment:
1. Please call it "JS stack trace". Except for JavaScript itself, we always abbreviate it as JS for all other components.
2. Please move this under "JS Extras" and above "JS Templates". The list sorted roughly alphabetically.